### DISCUSSION TITLE
VshareCloud-Project | 基于Lotus+Linux的开源数据备份工具，助力Filecoin应用场景落地

### DISCUSSION THREAD

**Discussion by egoleecode** [filecoin-project/community-china#205] [订单市场]
https://github.com/VshareCloud-Project/Filecoin-Storage-Tool
# Filecoin-Storage-Tool 
VshareCloud-CLI

>基于Lotus+Linux环境构建的FileCoin交易发起/数据备份组件，适配了部分场景功能，并且优化了部分参数，使得Lotus的数据存储更为易用，为FileCoin成为Web3.0数字世界基石打下基础。

### 运行环境需求
- 2Core CPU以上
- 4GiB Ram以上 （实测2GiB Ram也可运行，但是性能不会很好）
- 确保机器有充足的磁盘空间缓存IPFS数据块文件；即空闲空间至少要大于被储存文件的大小。
- 支持Systemd的Linux系统

**Comment by zhaobo443** [comment]
FIL联合封装，用户提供质押币，公司提供设备和运维，收益60％,公司扣出20％服务费和20%设备费，合同期1080天，540天质押币退还，t3天封装。可实地考察。V:zhaobo9308
